src/v/security/gssapi_authenticator.cc

```diff
@@ -0,0 +1,514 @@
+/*
```

Comment:
An overall comment, would it be possible to maybe document what the GSS calls are doing, or maybe point at the example code you used?  I was following it ok until the SSF portion.

Thanks :)

Reply:
I think it was this: https://github.com/cyrusimap/cyrus-sasl/blob/master/plugins/gssapi.c